azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -564,6 +583,49 @@ private void createPod(final int executionId) throws ExecutorManagerException {
     this.executorLoader.updateVersionSetId(executionId, versionSet.getVersionSetId());
   }
 
+  /**
+   * Items extracted from the podTemplate will be merged with the pod-spec. Merging criterion is
+   * such that, if the item in the already created pod-spec has the same name as of the item
```

Comment:
Nit: if the item, in the already created pod-spec, has the ....

Reply:
done